aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
+--------------------------------
+For server operating systems, use the **Add Roles and Features Wizard** via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
 
-1. Create an ASP.NET Core app using the ASP.NET Core **Web Application** template.
-2. In **Solution Explorer**, right-click the project and select **Publish**.
+.. image:: pubIIS/_static/rs.png
 
-.. image:: pubIIS/_static/p1.png
+On the **Role services** step, select the IIS role services you desire or accept the default role services provided.
 
-3. In the **Publish Web** dialog, on the **Profile** tab, select **File System**.
+.. image:: pubIIS/_static/role-services.png
 
-.. image:: pubIIS/_static/fs.png
+Proceed through the **Confirmation** step to enable the web server role and services.
 
-4. Enter a profile name. Click **Next**.
-5. On the **Connection** tab, you can change the publishing target path from the default *..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish* folder. Click **Next**.
-6. On the **Settings** tab, you can select the configuration, target runtime version, and publish options. Click **Next**.
+.. _unlock-handlers:
 
-The **Preview** tab shows you the publish path (by default, the same directory as the ".sln" solution file).
+Install the Microsoft ASP.NET Core Module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Deploy to IIS server
------------------------------------
+Install the Microsoft ASP.NET Core Module version 1.0 or higher on the server:
 
-#. Navigate to the publish folder (*..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish folder* in this sample).
-#. Copy the **approot** and **wwwroot** directories to the target IIS server. Note: MSDeploy is the recommended mechanism for deployment, but you can use Xcopy, Robocopy or another approach. For information on using `Web Deploy` see :doc:`iis-with-msdeploy`.
-#. In IIS Manager, create a new web site and set the physical path to **wwwroot**. You can click on **Browse *.80(http)** to see your deployed app in the browser.
+- `AspNetCoreModule x64 MSI Installer <http://download.microsoft.com/download/7/2/D/72DD68D5-7386-4A41-8B8F-E21779A69E33/aspnetcoremodule_x64_en.msi>`_
+- `AspNetCoreModule x86 MSI Installer <http://download.microsoft.com/download/7/2/D/72DD68D5-7386-4A41-8B8F-E21779A69E33/aspnetcoremodule_x86_en.msi>`_
 
-.. image:: pubIIS/_static/b8.png
+Install .NET Core on the Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-IIS server configuration
------------------------------------
+If you are planning to host all of your applications as self-contained applications on your server (i.e., you will not deploy portable applications that require installing .NET Core on the server), you may skip this step. For more information on the difference between portable applications and self-contained applications, see `Types of portability in .NET Core <https://github.com/blackdwarf/core-docs/blob/clidocs/docs/core-concepts/app-types.md>`_.
```

Comment:
However, we can't keep this blurb if the only way to get ANCM is via the bundle. They will be installing both ... it's just that when they load their self-contained apps calling their included executables, the shared host will not be triggered. Therefore, we can kill this whole bit.

Side Note: I really don't like this: I definitely **do not** want the shared framework on my production servers. Please I hope you are pushing forward on OSS'ing the ANCM. Those of us who feel as I should be able to compile our own ANCM and put that on our servers independently of the bundle for our self-contained-only apps scenarios.